cmd/iam-object-store.go

```diff
@@ -802,7 +802,11 @@ func (iamOS *IAMObjectStore) loadAllFromObjStore(ctx context.Context, cache *iam
 	// Store the newly populated map in the iam cache. This takes care of
 	// removing stale entries from the existing map.
 	cache.iamSTSAccountsMap = stsAccountsFromStore
-	cache.iamSTSPolicyMap = stsAccPoliciesFromStore
+
+	stsAccPoliciesFromStore.Range(func(k string, v MappedPolicy) bool {
+		cache.iamSTSPolicyMap.Store(k, v)
+		return true
+	})
```

Comment:
Why do it this way? It will not clear the existing entries in `cache.iamSTSPolicyMap` which may be deleted/stale.

Reply:
wont refresh clear the stale entries?

Reply:
Ah ok nvm, my misunderstanding. We need to merge here because it is already populated from line 757